### PR TITLE
added parameter escaping to ensure that parameters get sent to oc cor…

### PR DESCRIPTION
**Pull request by dipunm**
…rectly

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "oc-client-browser",
-  "version": "1.3.2",
+  "version": "1.3.4",
```
Comment (matteofigus) [inline package.json]: a note here, the version is changed when publishing. I'll revert this.

Comment (dipunm) [reply package.json]: Thanks sorry

Comment (dipunm) [reply package.json]: Wait. I thought I only bumped it by one... :confused:

Comment (matteofigus) [reply package.json]: no worries. Published 1.3.3

**Pull request merged**